python/lsst/cp/pipe/utils.py

```diff
@@ -20,12 +20,27 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 
-__all__ = ['PairedVisitTaskRunner', ]
+__all__ = ['PairedVisitListTaskRunner', 'SingleVisitListTaskRunner',
+           'NonexistentDatasetTaskDataIdContainer', 'parseCmdlineNumberString',
+           'countMaskedPixels', 'checkExpLengthEqual']
+
+import re
+import numpy as np
 
 import lsst.pipe.base as pipeBase
+import lsst.ip.isr as ipIsr
+import lsst.log as lsstLog
+
 
+def countMaskedPixels(maskedIm, maskPlane):
```

Comment:
No docstring. Also, we don't already have this functionality in `afw.image`?

Reply:
Not that I know of.